djangoProject/book/admin.py

```diff
@@ -0,0 +1,7 @@
+from django.contrib import admin
+from .models import Book
+
+class BookModelAdmin(admin.ModelAdmin):
+    list_display = [field.name for field in Book._meta.fields]
```

Comment:
nit: better to put all field names explicitly to see what actually is going to be rendered